src/main/java/org/ibp/api/rest/germplasm/GermplasmListResource.java

```diff
@@ -1,65 +0,0 @@
-
-package org.ibp.api.rest.germplasm;
-
-import java.util.List;
-
-import org.ibp.api.domain.germplasm.GermplasmListDetails;
-import org.ibp.api.domain.germplasm.GermplasmListSummary;
-import org.ibp.api.java.germplasm.GermplasmListService;
-import org.ibp.api.rest.ResourceURLLinkProvider;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-
-import com.wordnik.swagger.annotations.Api;
-import com.wordnik.swagger.annotations.ApiOperation;
-
-@Api(value = "Germplasm List Services")
-@RestController
-@RequestMapping(GermplasmListResource.URL)
-public class GermplasmListResource {
-
-	public static final String URL = "/germplasmList";
-
-	@Autowired
-	private GermplasmListService germplasmListService;
```

Comment:
If org.ibp.api.java.germplasm.GermplasmListService is not used anywhere else then we can also remove it.